src/ScrollingContainer.js

```diff
@@ -270,15 +266,41 @@ ScrollingContainer.prototype.initScrolling = function () {
         container.position[direction] = Math.round(Position[direction]);
 
         self.updateScrollBars();
-
     };
 
-    this._checkLimit = function (limitName) {
-        if (!(limitName in limitsReached)) {
-            limitsReached[limitName] = true;
-            self.emit(limitName + "Overflow");
+    this._checkLimits = function (direction) {
+        if (!this.scrolling && Math.round(Speed[direction]) === 0) {
+            return;
```

Comment:
Only check limits if the user is `scrolling`(dragging) or there is some `Speed` being applied.